kitsune/wiki/views.py

```diff
@@ -525,6 +525,20 @@ def review_revision(request, document_slug, revision_id):
     revision_contributors = list(set(
         based_on_revs.values_list('creator__username', flat=True)))
 
+    # Get Unreviewed Revisions
+    unreviewed_revision = list(Revision.objects.filter(
```

Comment:
Since this is multiple revisions, the name should be `unreviewed_revisions`, to denote that it is more than one item.